### PR TITLE
Remove unuse slice in registry

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
Slice Tokens is currently unused in registry.  I suggest that remove Tokens.  

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: registry/types.go [hunk 1]
```diff
@@ -12,8 +12,6 @@ type RepositoryData struct {
 	ImgList map[string]*ImgData
 	// Endpoints is a list of endpoints returned in X-Docker-Endpoints
 	Endpoints []string
-	// Tokens is currently unused (remove it?)
-	Tokens []string
```
Comment (AkihiroSuda) [inline registry/types.go]: If this change is correct, you need to update the comment (L8-L9) as well

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

**Comment by AkihiroSuda** [comment]
cc @aaronlehmann

File: registry/types.go (hunk 1 above)
Comment (yuexiao-wang) [inline registry/types.go]: Updated the comment.
Thanks

**Comment by aaronlehmann** [comment]
LGTM

**Review by dmcgowan — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**